javascript/chronometer.js

```diff
@@ -1,34 +1,50 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    if(callback) {
```

Comment:
what happens here if I don't have a callback function? Your program should work regardless (if the callback argument is defined, execute it; if not, then continue like usual)